integration/src/test/java/org/mule/test/integration/exceptions/OnErrorContinueTestCase.java

```diff
@@ -186,6 +185,28 @@ public void onErrorTypeMatchSeveral() throws Exception {
     assertThat(getPayloadAsString(result), is("false apt1 apt2"));
   }
 
+  @Test
+  public void onErrorTypeMatchNameWildcard() throws Exception {
+    Message result = flowRunner("onErrorTypeMatchNameWildcard").withPayload(true).run().getMessage();
+    assertThat(result, is(notNullValue()));
+    assertThat(getPayloadAsString(result), is("true apt1 apt2"));
+
+    result = flowRunner("onErrorTypeMatchSeveral").withPayload(false).run().getMessage();
+    assertThat(result, is(notNullValue()));
+    assertThat(getPayloadAsString(result), is("false apt1 apt2"));
+  }
+
+  @Test
+  public void onErrorTypeMatchNamespaceWildcard() throws Exception {
+    Message result = flowRunner("onErrorTypeMatchNamespaceWildcard").withPayload(true).run().getMessage();
+    assertThat(result, is(notNullValue()));
+    assertThat(getPayloadAsString(result), is("true apt1 apt2"));
+
+    result = flowRunner("onErrorTypeMatchSeveral").withPayload(false).run().getMessage();
```

Comment:
I think you meant to test with false the same flow.

Reply:
You're right